pkg/traces/traces.go

```diff
@@ -134,11 +136,17 @@ func newLogger(zapLevel zapcore.LevelEnabler, fmt logging.Format) *zap.Logger {
 		encoder = zaplogfmt.NewEncoder(config)
 	}
 
-	logger := zap.New(zapcore.NewCore(
-		encoder,
-		os.Stdout,
-		zapLevel,
-	), zap.AddCaller())
+	zapAdapter := zapadapter.New(l)
+
+	logger := zap.New(
+		zapcore.NewTee(
+			zapcore.NewCore(
+				encoder,
+				os.Stdout,
+				zapLevel,
+			),
+			zapAdapter.Core()),
+		zap.AddCaller())
```

Comment:
This keep the same definition of the previous logger, but tees it off to the zapadapter where we passed HookedLogger in.